data/process_pdb_dataset.py

```diff
@@ -121,9 +118,14 @@ def process_mmcif(
     processed_mmcif_path = os.path.join(mmcif_subdir, f'{mmcif_name}.pkl')
     processed_mmcif_path = os.path.abspath(processed_mmcif_path)
     metadata['processed_path'] = processed_mmcif_path
-    with open(mmcif_path, 'r') as f:
-        parsed_mmcif = mmcif_parsing.parse(
-            file_id=mmcif_name, mmcif_string=f.read())
+    try:
+        with open(mmcif_path, 'r') as f:
+            parsed_mmcif = mmcif_parsing.parse(
+                file_id=mmcif_name, mmcif_string=f.read())
+    except:
```

Comment:
Is this ever hit? I thought line errors get saved in the parsed_mmcif on line 130.